src/public/manageDomain.ts

```diff
@@ -0,0 +1,173 @@
+const envList = helper.getElement('.envList')
+const environmentVariables: EnvironmentItem[] = []
+const fullDomain = window.location.pathname.split('/').pop()
+document.getElementById('full-domain').innerText = fullDomain
+
+class EnvironmentItem {
+  name: string
+  value: string
+  isValid = false
+
+  private nameInputElement: HTMLInputElement
+  private valueInputElement: HTMLInputElement
+
+  private readonly IS_INVALID = 'is-invalid'
+  private readonly LETTER_NUMBER_REGEX = /^[A-Z0-9_]+$/g
+  private readonly ELEMENT_HTML = `
+    <div class="input-group">
+      <input 
+        type="text"
+        aria-label="Name"
+        class="form-control text-uppercase text-monospace name-input ${this.IS_INVALID}"
```

Comment:
By default, the element is supposed to have `is-invalid` class?

Reply:
It starts empty, and if the value is sent to the constructor then it gets marked as valid. This is to prevent sending variables with no name, which makes no sense.

Reply:
I guess it makes sense haha
But, shouldn't the constructor just handle marking it valid or invalid all together?